colibre/registration.py

```diff
@@ -143,9 +145,46 @@
 total_dust_mass = total_dust_fraction * catalogue.masses.m_gas
 name = f"$M_{{\\rm dust}}${dust_frac_error}"
 total_dust_mass.name = name
-nonmetal_frac = 1. - catalogue.apertures.zmet_gas_sf_100_kpc
+total_dust_fraction.name = f"$\\mathcal{{DTG}}${dust_frac_error}"
+total_dust_mass = total_dust_fraction * catalogue.masses.m_gas
+total_dust_mass.name = f"$M_{{\\rm dust}}${dust_frac_error}"
+dust_to_metals = total_dust_fraction / metal_frac
+dust_to_metals.name = f"$\\mathcal{{DTM}}${dust_frac_error}"
+dust_to_stars = total_dust_mass / catalogue.apertures.mass_star_100_kpc
+dust_to_stars.name = f"$M_{{\\rm dust}}/M_*${dust_frac_error}"
 
+nonmetal_frac = 1. - catalogue.apertures.zmet_gas_sf_100_kpc
+    
 setattr(self, f"total_dust_masses_100_kpc", total_dust_mass)
+setattr(self, f"dust_to_metal_ratio_100_kpc", dust_to_metals)
+setattr(self, f"dust_to_gas_ratio_100_kpc", total_dust_fraction)
+setattr(self, f"dust_to_stellar_ratio_100_kpc", dust_to_stars)
+
+# Get depletion properties
+try:
+    gas_mass = catalogue.masses.m_gas
+    H_frac = getattr(catalogue.element_mass_fractions, "element_0")
+    O_frac = getattr(catalogue.element_mass_fractions, "element_4")
+    
+    o_abundance = unyt.unyt_array(
+        12+np.log10(O_frac/(16*H_frac)), "dimensionless")
+    o_abundance.name = "$12+\\log_{10}({{\\rm O/H}})$" 
+    setattr(self, "gas_o_abundance", o_abundance)
+    
+except AttributeError:
+    # We did not produce these quantities.
+    setattr(
+        self,
+        "gas_o_abundance",
+        unyt.unyt_array(
+            catalogue.masses.m_gas**0,
```

Comment:
not sure that this has the correct units for the 'fallback' array.

Reply:
the `**0` makes it dimensionless